exercises/simple-linked-list/src/test/scala/SimpleLinkedListTest.scala

```diff
@@ -0,0 +1,56 @@
+import org.scalatest.{FlatSpec, Matchers}
+
+class SimpleLinkedListTest extends FlatSpec with Matchers {
```

Comment:
Some of the tests are FunSuite some are FlatSpec. There really isn't any consistency, both are used.